packages/contracts/src/networks.ts

```diff
@@ -0,0 +1,480 @@
+  },
+  {
+    name: 'polygon_zkevm_goerli',
+    chainId: BigInt(1442),
+    rpcUrl: () => process.env.POLYGON_ZKEVM_TESTNET_URL!,
+    etherscan: {
+      apiURL: 'https://api-testnet-zkevm.polygonscan.com/api',
+      browserURL: 'https://testnet-zkevm.polygonscan.com',
+    },
+    currency: 'ETH',
+    dripSize: '0.15',
+    requiredEnvVariables: ['POLYGON_ZKEVM_TESTNET_URL'],
+  },
+  {
+    name: 'avalanche',
+    chainId: BigInt(43114),
+    rpcUrl: () =>
+      `https://avalanche-mainnet.infura.io/v3/${process.env.INFURA_API_KEY}`,
+    etherscan: {
+      apiURL: 'https://api.snowtrace.io/api',
+      browserURL: 'https://snowtrace.io/',
+    },
+    currency: 'AVAX',
+    dripSize: '1',
+    requiredEnvVariables: ['INFURA_API_KEY'],
+  },
+  {
+    name: 'avalanche_fuji',
+    chainId: BigInt(43113),
+    rpcUrl: () =>
+      `https://avalanche-fuji.infura.io/v3/${process.env.INFURA_API_KEY}`,
+    etherscan: {
+      apiURL: 'https://api-testnet.snowtrace.io/api',
+      browserURL: 'https://testnet.snowtrace.io/',
+    },
+    currency: 'AVAX',
+    dripSize: '1',
+    requiredEnvVariables: ['INFURA_API_KEY'],
+  },
+  {
+    name: 'fantom',
+    chainId: BigInt(250),
+    rpcUrl: () => process.env.FANTOM_MAINNET_RPC_URL!,
+    etherscan: {
+      apiURL: 'https://api.ftmscan.com/api',
+      browserURL: 'https://ftmscan.com',
+    },
+    currency: 'FTM',
+    dripSize: '1',
+    requiredEnvVariables: ['FANTOM_MAINNET_RPC_URL'],
+  },
+  {
+    name: 'fantom_testnet',
+    chainId: BigInt(4002),
+    rpcUrl: () => process.env.FANTOM_TESTNET_RPC_URL!,
+    etherscan: {
+      apiURL: 'https://api-testnet.ftmscan.com/api',
+      browserURL: 'https://testnet.ftmscan.com',
+    },
+    currency: 'FTM',
+    dripSize: '1',
+    requiredEnvVariables: ['FANTOM_TESTNET_RPC_URL'],
+  },
+  {
+    name: 'base',
+    chainId: BigInt(8453),
+    rpcUrl: () =>
+      `https://base-mainnet.g.alchemy.com/v2/${process.env.ALCHEMY_API_KEY}`,
+    etherscan: {
+      apiURL: 'https://api.basescan.org/api',
+      browserURL: 'https://basescan.org/',
+    },
+    currency: 'ETH',
+    dripSize: '0.025',
+    requiredEnvVariables: ['ALCHEMY_API_KEY'],
+  },
+  {
+    name: 'base_sepolia',
+    chainId: BigInt(84532),
+    rpcUrl: () =>
+      `https://base-sepolia.g.alchemy.com/v2/${process.env.ALCHEMY_API_KEY}`,
+    etherscan: {
+      apiURL: 'https://base-sepolia.blockscout.com/api',
+      browserURL: 'https://base-sepolia.blockscout.com/',
+    },
+    currency: 'ETH',
+    dripSize: '0.15',
+    requiredEnvVariables: ['ALCHEMY_API_KEY'],
+  },
+  // {
+  //   name: 'celo',
+  //   chainId: BigInt(42220),
+  //   rpcUrl: () =>
+  //     `https://celo-mainnet.infura.io/v3/${process.env.INFURA_API_KEY}`,
+  //   etherscan: {
+  //     apiURL: 'https://api.celoscan.io/api',
+  //     browserURL: 'https://celoscan.io/',
+  //   },
+  //   currency: 'CELO',
+  //   dripSize: '1',
+  //   requiredEnvVariables: ['INFURA_API_KEY'],
+  // },
```

Comment:
I was originally going to also add support for all the networks requested by mean finance as part of this PR. I chose not to, though, b/c I don't want this PR to be blocked by all the related tasks (deploying contracts, getting etherscan keys, handling funds on all the networks, etc).